doc/design.md

```diff
@@ -491,6 +491,31 @@ Covers:
 
 Needs: impl, itest
 
+## Workarounds
+
+### Log Rotation Workaround Criteria
+`dsn~log-rotation-workaround-criteria~1`
+
+WorkaroundManager applies the `LogRotatinWorkaround` if the exasol version is 7.0.x or lower.
+
+
+Covers:
+
+* `req~log-rotation-workaround~1`
+
+Needs: impl, utest
```

Comment:
maybe also here itest missing

Reply:
For this one a unit test is good enough.